### PR TITLE
feat(system-server): add system-server to buildroot

### PR DESCRIPTION
**Pull request by fsinapi**
# Overview

In order to include the system-server project in the OT-2 buildroot build, we need a Config.in and a systemd system file.

# Test Plan

Make a companion buildroot PR [here](https://github.com/Opentrons/buildroot/pull/194) that adds the system-server project, and then:

- [x] Ensure that the two branches (buildroot & monorepo) build an update image
- [x] Upload the image to an OT-2
- [x] Check that systemctl is running the `system-server` application
- [x] Open a python repl and verify that `PyJWT` is available
- [x] Confirm that a system server version is included in the version file

# Changelog

- Added buildroot configuration files for the system-server project

# Review requests

# Risk assessment

Medium. This is going to be added into the OT-2 image, so it's important to verify that the robot still functions with this running the background for now.

### CODE REVIEW

File: system-server/Config.in [hunk 1]
```diff
@@ -0,0 +1,10 @@
+config BR2_PACKAGE_PYTHON_OPENTRONS_SYSTEM_SERVER
+  bool "python-opentrons-robot-server"
```
Comment (sfoster1) [inline system-server/Config.in]:
```suggestion
  bool "python-opentrons-system-server"
```

**Review by sfoster1 — CHANGES_REQUESTED** [review]
A small change needed - and also this won't get built unless you add it to the defconfig in buildroot

**Review by sfoster1 — APPROVED** [review]
Looks good to me pending testing.

**Comment by vegano1** [comment]
config-wise this looks good

**Review by vegano1 — APPROVED** [review]

**Comment by fsinapi** [comment]
Added systemd notification integration and tested on an OT-2. Confirmed that the notification gets sent to systemd.

**Review by sfoster1 — APPROVED** [review]
Looks good!

**Pull request merged**